modules/core/test/v2/unit/coins/abstractEthCoin.ts

```diff
@@ -0,0 +1,68 @@
+import * as _ from 'lodash';
```

Comment:
Could you add 

```
/**
 * @prettier
 */
```

or manually run prettier on this? 8 space indents is a bit too much.